src/react-apps/applications/receipt/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Maybe this file should be in the /__tests__ folder?

Reply:
Fixed.